src/classlibnative/bcltype/diyfp.h

```diff
@@ -0,0 +1,80 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+//
+// File: diyfp.h
+//
+
+//
+
+#ifndef _DIYFP_H
+#define _DIYFP_H
+
+#include <clrtypes.h>
+
+// An exteneded floating-point data structure.
+// It defines a 64-bit significand and a 32-bit exponent, 
+// which is EXTENDED compare to IEEE double precision floating-point number. 
+class DiyFp
```

Comment:
It would be helpful if this was more significantly documented.

What is it used for, why it is needed, and the reasoning for the specific format (96-bits, 64-bit significand, 32-bit exponent).

Most of it is not immediately obvious and will be initially confusing to anyone else coming to touch the code later.

Reply:
OK will do.